pom.xml

```diff
@@ -1,40 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-parent</artifactId>
+        <version>2.0.1</version>
+    </parent>
     <groupId>com.vaadin</groupId>
-    <artifactId>vaadin-parent</artifactId>
-    <version>1.1.0</version>
-  </parent>
-  <groupId>com.vaadin</groupId>
-  <artifactId>vaadin-archetypes</artifactId>
-  <version>8.1-SNAPSHOT</version>
-  <packaging>pom</packaging>
+    <artifactId>vaadin-archetypes</artifactId>
+    <version>13.0.1</version>
```

Comment:
13.0.2 was just out
```suggestion
    <version>13.0.2</version>
```

Reply:
BTW. This version should bi 13.0.2-SNAPSHOT in the SCM, until the actual archetypes are built.

Reply:
Done